platform_cli/helpers.py

```
@@ -11,6 +11,50 @@
+        affected_packages = set()
+        for changed_file in changed_files:
+            for package_name, package_path in all_packages.items():
+                try:
+                    changed_file.resolve().relative_to(package_path.resolve())
+                    affected_packages.add(package_name)
+                    break
+                except ValueError:
+                    continue
+
+        return list(affected_packages)
+    except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)